tests/playwright/src/model/preferences-extension-ai-lab-page.ts

```diff
@@ -0,0 +1,49 @@
+import type { Locator, Page } from '@playwright/test';
+import type { NavigationBar } from '@podman-desktop/tests-playwright';
+import { expect as playExpect, PreferencesPage } from '@podman-desktop/tests-playwright';
+
+export class ExtensionAILabPreferencesPage extends PreferencesPage {
+  public static readonly tabName = 'Extension: AI Lab';
+  readonly heading: Locator;
+
+  constructor(page: Page) {
+    super(page);
+    this.heading = page.getByRole('heading', { name: ExtensionAILabPreferencesPage.tabName });
+  }
+
+  async waitForLoad(): Promise<void> {
+    await playExpect(this.heading).toBeVisible();
+  }
+
+  public static async openAILabPreferences(
```

Comment:
Same as for openAIDashboard. this does not belong here.

Reply:
corrected